umm-spec-lib/src/cmr/umm_spec/models/umm_service_models.clj

```diff
@@ -168,9 +168,6 @@
    ;; This is the long name of the service organization.
    LongName
 
-   ;; Uuid of the service organization.
```

Comment:
Is github going a bad job of the diffs here? Looks like some fields got dropped that I wasn't expecting? (maybe they've just shifted elsewhere in the file, and I'm not seeing where ...)

Reply:
Its possible that this was dropped. I downloaded the schema v1.1 from the repo and not all changes were listed in the ticket.  But if its changed in the schema then it should be changed in this PR.